.github/workflows/pull-compliance-docsignore.yml

```diff
@@ -7,7 +7,32 @@ on:
       - "*.md"
 
 jobs:
-  compliance-docs:
```

Comment:
`compliance-docs:` is still needed, for builds that won't need any docs to be built.

Reply:
But it has been defined in `pull-compliance-docs.yml`?

Reply:
It will be ignored because:
```
on:
  pull_request:
    paths:
      - "docs/**"
      - "*.md"
```
in that file, so it needs to be defined in this file too.

Reply:
done.